core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataSchema.java

```diff
@@ -263,6 +263,8 @@ public static void validateDirCol(String dirName) {
        */
       public static final String OPID_QUAL = "opid";
       public static final ColumnFQ OPID_COLUMN = new ColumnFQ(NAME, new Text(OPID_QUAL));
+      public static final String NEXT_OPID_QUAL = "opid_next";
```

Comment:
I'm not set on `opid_next`. In fact, I'm not sure that `opid` is exactly correct either. A `opid_next` entry will cause a hosted tablet to be unhosted. A `opid` entry will prevent a tablet from being assigned or hosted. The value that gets put into this column is the TabletOperationType (SPLITTING, MERGING, DELETING) and the Fate Tx Id. This is only used for Fate operations that are destructive to the tablet, not all Fate operations. I'm wondering if the naming of the column should be more descriptive to convey this.

Reply:
We could also call call `opid` a `modification` and `opid_next` a `pending_modification`. The value in the entry has the modification type and fate tx id, so we don't need `id` in the name.